sched/definitions.go

```diff
@@ -17,3 +17,33 @@ type Task struct {
 	Command    []string // Argv to run
 	SnapshotId string
 }
+
+type JobState int
+
+const (
+	// An unambiguous 0-value.
+	UNKNOWN JobState = iota
+	// Waiting to run.
+	PENDING
+	// Running
+	RUNNING
+	// Done
+	DONE
+)
+
+type JobStatus struct {
+	ID    string
+	State JobState
+}
+
+func PendingStatus(jobID string) (s JobStatus) {
+	s.ID = jobID
+	s.State = PENDING
+	return s
+}
+
+func UnknownStatus(jobID string) (s JobStatus) {
```

Comment:
There should be at some point, but the Queue will never deal with jobs in those status (because the Queue never runs or knows about done jobs). I can remove them.

That's a good point about should they be "MakeFoo". I guess I think it's kind of different because it's not JobStatus, it's RunningStatus, so we don't need the Make to differentiate the name from the struct type. I think it's nice to have the shorter method names, but I don't feel at all strongly.